src/main/java/org/broadinstitute/hellbender/tools/walkers/bqsr/AnalyzeCovariates.java

```diff
@@ -477,13 +477,7 @@ private File resolveCsvFile() {
         if (csvFile != null)  {
             return csvFile;
         } else {
-            try {
-              final File result = File.createTempFile("AnalyzeCovariates", ".csv");
-              result.deleteOnExit();
-              return result;
-            } catch (IOException e) {
-                throw new UserException("Could not create temporary Csv file",e);
-            }
+            return BaseTest.createTempFile("AnalyzeCovariates", ".csv");
```

Comment:
Keeping `BaseTest.createTempFile()` in place as a trivial pass-through to the `IOUtils` one will make it easier to port test classes from the old GATK -- I vote we keep it for now.